homeassistant/components/nfandroidtv/notify.py

```
@@ -178,18 +186,22 @@ def send_message(self, message: str, **kwargs: Any) -> None:
                         translation_key="invalid_notification_icon",
                         translation_placeholders={"type": type(icondata).__name__},
                     )
-        self.notify.send(
-            message,
-            title=title,
-            duration=duration,
-            fontsize=fontsize,
-            position=position,
-            bkgcolor=bkgcolor,
-            transparency=transparency,
-            interrupt=interrupt,
-            icon=icon,
-            image_file=image_file,
-        )
+
+        try:
+            self.notify.send(
+                message,
+                title=title,
+                duration=duration,
+                fontsize=fontsize,
+                position=position,
+                bkgcolor=bkgcolor,
+                transparency=transparency,
+                interrupt=interrupt,
+                icon=icon,
+                image_file=image_file,
+            )
+        except ConnectError:
+            _LOGGER.warning("Failed to connect to host: %s", self.host)
```

Instead, let's raise a `HomeAssistantError` instead of silently failing